app/graphql/mutations/create_commit_feedback.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Mutations
+  class CreateCommitFeedback < BaseMutation
+    include CompassUtils
+
+    field :status, String, null: false
+
+    argument :label, String, required: true, description: 'repo or project label'
+    argument :level, String, required: false, description: 'repo or community', default_value: 'repo'
+    argument :commit_feedback_input, Input::CommitFeedbackInput, required: true, description: 'repo extension'
+
+    def resolve(label: nil, level: 'repo', commit_feedback_input: nil)
+      label = ShortenedLabel.normalize_label(label)
+      login_required!(context[:current_user])
+
+      git_indexer, repo_urls = select_idx_repos_by_lablel_and_level(label, level, GiteeGitEnrich, GithubGitEnrich)
+      commit_data = git_indexer.fetch_commit_one_by_hash(repo_urls, commit_feedback_input[:commit_hash])
+      raise GraphQL::ExecutionError.new I18n.t('commit_feedback.invalid_commit_hash') if commit_data.nil?
```

Comment:
@lishengbao And I don't see a change in the i18n file, but it is used inside the code file, has the relevant file been submitted?